content/client-lib-development-guide/features.textile

```diff
@@ -148,17 +148,19 @@ h3(#rest-auth). Auth
 ** @(RSA9g)@ A valid HMAC is created using the key secret  to sign the @TokenRequest@ so that it can be used by any client to request a token without having or exchanging any secrets
 ** @(RSA9i)@ Adheres to all requirements in @RSA8@ relating to @TokenParams@
 * @(RSA10)@ @Auth#authorise@ function:
```

Comment:
From https://github.com/ably/wiki/issues/68#issuecomment-188697130 :

> However, given where we are, I think we should get consistency in `initialized` everywhere, and worry about `authorize` perhaps in the 0.9 spec.

This might be a good opportunity? (Though should keep `authorise` as an alias with a deprecated warning till 1.0 at least)